chpl/chpl-service/src/main/java/gov/healthit/chpl/scheduler/job/report/attestation/AttestationReportCreatorJob.java

```diff
@@ -138,29 +141,64 @@ protected void doInTransactionWithoutResult(TransactionStatus status) {
                 }
             }
 
-            private void updateCountsBasedOnChangeRequestStatus(ChangeRequest cr, AttestationReport report) {
-                if (cr == null) {
-                       report.setNoSubmissionCount(report.getNoSubmissionCount() + 1);
-                   } else {
-                       switch (cr.getCurrentStatus().getChangeRequestStatusType().getName()) {
-                       case "Accepted":
-                           report.setApprovedCount(report.getApprovedCount() + 1);
-                           break;
-                       case "Pending Developer Action":
-                           report.setPendingDeveloperActionCount(report.getPendingDeveloperActionCount() + 1);
-                           break;
-                       case "Pending ONC-ACB Action":
-                           report.setPendingAcbActionCount(report.getPendingAcbActionCount() + 1);
-                           break;
-                       default:
-                           break;
-                       }
-                   }
-            }
+
         });
         LOGGER.info("********* Completed Attestation Report Creator job. *********");
     }
 
+    private void updateCountsBasedOnChangeRequestStatus(ChangeRequest cr, Developer developer, AttestationReport report) {
+        if (cr == null) {
+               report.setNoSubmissionCount(report.getNoSubmissionCount() + 1);
+               report.getDevelopersWithNoSubmissionAttestations().add(DeveloperAttestationStatus.builder()
+                       .developer(developer)
+                       .build());
+       } else {
+           switch (cr.getCurrentStatus().getChangeRequestStatusType().getName()) {
+           case "Accepted":
```

Comment:
Instead of the hardcoded string here, I wonder if you could use this property for the status ID that I've seen us use elsewhere. 
`@Value("${changerequest.status.accepted}")`

Reply:
Cannot `switch` using `Long` value.  Changed to use an enum.